src/nextjs/templates.ts

```diff
@@ -711,3 +711,139 @@ export default function RootLayout({
   )
 }
 `;
+
+export function getAiRulesFileContent(): string {
+  return `These examples should be used as guidance when configuring Sentry functionality within a project.
+
+# Exception Catching
+
+- Use \`Sentry.captureException(error)\` to capture an exception and log the error in Sentry.
+- Use this in try catch blocks or areas where exceptions are expected
+
+# Tracing Examples
+
+- Spans should be created for meaningful actions within an applications like button clicks, API calls, and function calls
+- Ensure you are creating custom spans with meaningful names and operations
+- Use the \`Sentry.startSpan\` function to create a span
+- Child spans can exist within a parent span
+
+## Custom Span instrumentation in component actions
+
+- The \`name\` and \`op\` properties should be meaninful for the activities in the call.
+- Attach attribute based on relevant information and metrics from the request
+
+\`\`\`javascript
+function TestComponent() {
+  const handleTestButtonClick = () => {
+    // Create a transaction/span to measure performance
+    Sentry.startSpan({ 
+      op: "ui.click", 
+      name: "Test Button Click" 
+    }, (span) => {
+       
+      const value = "some config"
+      const metric = "some metric"
+      
+      // Metrics can be added to the span
+      span.setAttribute("config", value)
+      span.setAttribute("metric", metric)
+      
+      doSomething();
+    });
+  };
+
+  return (
+    <button 
+      type="button" 
+      onClick={handleTestButtonClick}
+    >
+      Test Sentry
+    </button>
+  );
+}
+\`\`\`
+
+## Custom span instrumentation in API calls
+
+- The \`name\` and \`op\` properties should be meaninful for the activities in the call.
+- Attach attributes based on relevant information and metrics from the request
+
+\`\`\`javascript
+async function fetchUserData(userId) {
+  return Sentry.startSpan(
+    {
+      op: "http.client",
+      name: \`GET /api/users/\${userId}\`,
+    },
```

Comment:
IMHO manually created spans don't need to contain a well curated op. As per my comment above, I think `http.client` wouldn't be a good choice because a `http.client` span should only describe the duration of the fetch call. Also, since we can actually parameterize the route fetch request here (i.e. show the :userId parameter instead of the actual id (e.g. 1234)) we should leverage that. 

```suggestion
    {
      name: \'fetch /api/users/:userId',
    },
```